scripts/kconfig/confdata.c

```diff
@@ -636,6 +636,55 @@ static struct conf_printer kconfig_printer_cb =
 	.print_comment = kconfig_print_comment,
 };
 
+/*
+ * Rust configuration printer
+ *
+ * This printer is used when generating the resulting rustc configuration
+ * after kconfig invocation and `defconfig` files.
+ *
+ */
+static void
```

Comment:
Please follow the same convention as in the rest of the file (same line for the signature).

Reply:
I saw the file has different styles, so it does not matter.